doc/source/whatsnew/v1.1.0.rst

```diff
@@ -54,6 +54,7 @@ Other API changes
 
 - :meth:`Series.describe` will now show distribution percentiles for ``datetime`` dtypes, statistics ``first`` and ``last``
   will now be ``min`` and ``max`` to match with numeric dtypes in :meth:`DataFrame.describe` (:issue:`30164`)
+- Added :meth:`DataFrame.value_counts` (:issue:`5377`)
```

Comment:
add a doc-link to the section in the docs where you showcase this (need to add)

Reply:
add docs near where we talk about Series.value_counts

Reply:
A bit confused by this one. Where should I showcase the method and how to link to it?